src/tests/MyProfile.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import MyProfile from '../components/myProfile';
+import store from '../redux/configureStore';
+
+it('matches snapshot', () => {
+  const tree = renderer.create(
+    <Provider store={store}>
+      <MyProfile />
+    </Provider>,
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ Required ] Great job adding `snap shot` tests to your project, however, I noticed the tests are failing when I run them. Please make sure that all your tests are passing, please see the screen shot below for more details on this and rectify.

![Screenshot from 2023-02-09 00-01-22](https://user-images.githubusercontent.com/16392046/217650534-16a2980c-e096-4e3e-ad15-213f839930c6.png)

Reply:
Hi @Felix45, we just fixed it


<img width="566" alt="Screenshot 2023-02-09 at 12 23 28 PM" src="https://user-images.githubusercontent.com/110017001/217752092-2afd1c46-222b-4841-ae5f-2dd2953a6481.png">